File: src/test/java/organizing/data/encapsulateFields.java
package organizing.data;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class encapsulateFields {
    @Test
    public void harryPotter() {
        Wizard harryPotter = new Wizard("Harry Potter");
        harryPotter.life = 100;
        harryPotter.magicStick = "Bâton de la Mort";
        harryPotter.weapon = "None";

        assertThat(harryPotter.life).isEqualTo(100);
    }
}

File: src/scala/src/test/scala/organizing/data/encapsulateFields.scala
package org.ythirion.refactoring.journey
package organizing.data

import org.scalatest.funsuite.AnyFunSuite

class encapsulateFields extends AnyFunSuite {
  test("Harry Potter") {
    val harryPotter = Wizard("Harry Potter",
      Some("Bâton de la Mort"),
      Some("None"),
      100)

    assert(harryPotter.life == 100)
  }
}
